torchtext/nn/modules/multiheadattention.py

```diff
@@ -223,9 +223,19 @@ def forward(self,
         Args:
             query, key, value (Tensors): sequence to be projected
 
+        Examples::
+            >>> from torchtext.nn import InProjContainer
+            >>> embed_dim, bsz = 10, 64
+            >>> in_proj_container = InProjContainer(torch.nn.Linear(embed_dim, embed_dim),
+                                                    torch.nn.Linear(embed_dim, embed_dim),
+                                                    torch.nn.Linear(embed_dim, embed_dim))
+            >>> q = torch.rand((5, bsz, embed_dim))
+            >>> k = v = torch.rand((6, bsz, embed_dim))
+            >>> q, k, v = in_proj_container(q, k, v)
+
         Shape:
-            - query, key, value: :math:`(S, N, E)`
-            - Output: :math:`(S, N, E)`.
+            - query, key, value: :math:`(S, N, E)`, :math:`(S, N, E)`, :math:`(S, N, E)`
```

Comment:
I'd consider this misleading, because it entirely depends on the module that the user passes in and by default the linear layer supports broadcasting and any size or shape applicable to matrix multiplication.

Reply:
If this is about the default, i think we should mention that and also refer to the documentation of linear.

Reply:
In fact, let's just write out the code we have as documentation since it's super basic for now. As a follow-up task, maybe it's time to look into using torch.bmm when applicable.